pkg/kudoctl/util/repo/repo_framework.go

```diff
@@ -92,8 +85,9 @@ func (r *FrameworkRepository) GetPackage(bundleName string) (*PackageCRDs, error
 	return fvPackage, nil
```

Comment:
doing it similar like in the `DownloadIndexFile()` method?
```suggestion
        fvPackage, err := ReadTarGzPackage(resp)
	return fvPackage, err
```

Reply:
I think it was done like this to have additonal error information: `unable to parse the package. Errors are:`. But I can change this if you want.